cmd/storage-provisioner/main.go

```diff
@@ -25,6 +25,8 @@ import (
 	"k8s.io/minikube/pkg/storage"
 )
 
+var pvDir = "/tmp/hostpath-provisioner"
```

Comment:
This location feels weird. Is this a placeholder?

Reply:
It's the paths that we are currently using, it gets bind-mounted/symlinked onto the persistent disk.

See https://minikube.sigs.k8s.io/docs/reference/persistent_volumes/  (it also uses `/tmp/hostpath_pv`)